tests/cephadm/test_cluster_view.py

```diff
@@ -0,0 +1,108 @@
+from json import loads
+
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from utility.log import Log
+
+log = Log(__name__)
+
+DEFAULT_CEPH_DIR = "/etc/ceph/"
+DEFAULT_CONF_PATH = "/etc/ceph/ceph.conf"
+
+
+class ClusterViewError(Exception):
+    pass
+
+
+def verify_mon_service(host):
+    """
+    Verify MON service is up
+    Args:
+        host(ceph): Installer host
+    """
+    timeout, interval = 20, 6
+    for w in WaitUntil(timeout=timeout, interval=interval):
+        out = CephAdm(host).ceph.orch.ls(service_type="mon", format="json")
+        configs = loads(out)
+        for config in configs:
+            running = config.get("status", {}).get("running")
+            size = config.get("status", {}).get("size")
+        if running == size:
+            log.info("All MON services are up and running")
```

Comment:
These logs are not present here: http://magna002.ceph.redhat.com/cephci-jenkins/aramteke/Automation/Ceph-83574667/Verify_Cluster_View_0.log

Is the attached log latest?

Reply:
I'll attach full logs again.